include/crocoddyl/core/numdiff/actuation.hpp

```diff
@@ -1,7 +1,7 @@
 ///////////////////////////////////////////////////////////////////////////////
 // BSD 3-Clause License
 //
-// Copyright (C) 2018-2020, University of Edinburgh
+// Copyright (C) 2018-2020, LAAS-CNRS, University of Edinburgh
```

Comment:
Please do not put LAAS-CNRS as first copy holder. I will fix for you this time.

Note that this file (and others in the repo) have been developed in UoE.